presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -778,6 +794,11 @@ Procedures
     Unregisters given, existing partition in the metastore for the specified table.
     The partition data is not deleted.
 
+HDFS Erasure Coding
+-------------------
+
+Hive connector supports Hadoop 3's HDFS Erasure Coding.
```

Comment:
The Hive connector

Reply:
also .. should we add a link to some Erasure Coding docs? Or some more info on what that means?

Reply:
adding link